Sources/MongoSwift/BSON/Overwritable.swift

```diff
@@ -43,8 +43,9 @@ extension Double: Overwritable {
 
 extension Decimal128: Overwritable {
     internal func writeToCurrentPosition(of iter: DocumentIterator) throws {
-        var encoded = try Decimal128.toLibBSONType(self.data)
-        bson_iter_overwrite_decimal128(&iter.iter, &encoded)
+        // TODO: avoid this copy via withUnsafePointer once swift 4.1 support is dropped (SWIFT-284)
```

Comment:
Yeah, I'll just wait for the merge then fix them up in this PR

Reply:
I think this spot still needs to be updated.

Reply:
whoops, fixed.

I have to cast to a mutable pointer here due to a C driver oversight. I filed [CDRIVER-3069](https://jira.mongodb.org/browse/CDRIVER-3069) for it, but I doubt it will be fixed any time soon / ever.